schema.graphql

```diff
@@ -139,6 +139,18 @@ type AuthenticationStatus {
   invitation: Invitation
 }
 
+type BooleanWidget implements Component {
+  checked: Boolean
```

Comment:
idk if this is the wording we want or not

Reply:
This is fine